hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableInputFormat.java

```diff
@@ -114,9 +114,15 @@
     }
     this.readCommittedScope = getReadCommitted(job, identifier);
     LoadMetadataDetails[] loadMetadataDetails = readCommittedScope.getSegmentList();
-
-    SegmentUpdateStatusManager updateStatusManager =
-        new SegmentUpdateStatusManager(carbonTable, loadMetadataDetails);
+    String updateDeltaVersion = job.getConfiguration().get("updateDeltaVersion");
```

Comment:
Please make `updateDeltaVersion` a constant and add comment to describe it

Reply:
If version is supported in hadoop input format level, then hive and presto can also leverage this information when reading carbon, right?

Reply:
ok. Yes it can be leveraged from presto as well